### DISCUSSION TITLE
Remove unused functions and declared functions?

### DISCUSSION THREAD

**Discussion by Vulpine05** [BOINC/boinc#5843] [Q&A] [answered]
This question was brought up by @davidpanderson in (https://github.com/BOINC/boinc/issues/5841#issuecomment-2400975811) that if a function is not used anywhere, it should be removed.  I have found a few other functions that are declared in the header file  with no inline code, and the function is not in the .cpp file.

I can create a PR to remove any unused function declarations that I have found, but I wanted to double check that this is okay to do.  I didn't dig through the history of the code to see why there is a header with no code.

**Comment by davidpanderson** [comment, marked as answer]
Yes, please!
